docs/pages/eas/metadata/index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: EAS Metadata
+sidebar_title: Introduction
+hideTOC: true
+---
+
+import { BoxLink } from '~/ui/components/BoxLink';
+import { Callout } from '~/ui/components/Callout';
+import { Terminal } from '~/ui/components/Snippet';
+
+<Callout type="warning">
+  EAS Metadata is in beta and subject to breaking changes.
+</Callout>
+<br />
+
+**EAS Metadata** is a tool that helps submit app store information to publish your app.
+
+You need to provide a lot of information to multiple app stores before your users can use your app. This information is often about complex topics that don't apply to your app. You have to start a lengthy review process after providing the information. When the reviewer finds any issues in the information you provided, you need to restart this process.
+
+## Store config
+
+EAS Metadata aims to make this easier by using a [**store.config.json**](./config.md#static-store-config) file. You can provide all information in this store instead of going through multiple forms in the app store dashboards. When it's time to update the app stores, you can push the store config to the app stores.
```

Comment:
This header is strange because it punctuates the previous paragraph, then the next paragraph continues by saying "make this easier".

Reply:
Don't say "aims to make this easier", the feature does or doesn't do things.

Reply:
I'd add a part about how you can store your metadata locally in git.